### ISSUE TITLE
3DMark03 issues on VirtualBox

### ISSUE THREAD

**Issue by landandsky** [JHRobotics/softgpu#101]
I get these errors when trying to run and install the benchmark, any idea of what could be happening?

windows 98 se vgpu10 with the latest driver, on an intel i7 processor.

![1](https://github.com/user-attachments/assets/7e16928e-f17b-4b21-822d-f52c8ca8d870)
![2](https://github.com/user-attachments/assets/25e41801-3e40-438e-99b3-161c89f50a1c)

**Comment by ride2x4** [comment]
Is it 3dMark03 build 340 you try to install? Newer builds don't support Win98.

**Comment by landandsky** [comment]
the .exe itself is named 3DMark03_v360_1901, does that help?

**Comment by JHRobotics** [comment]
Yes, this is 3DMark03 version 3.6.0 which not supports Windows 98. Last version working in Win9x is 3.5.0, you can [download it from me](files.emulace.cz/demos/3dmark03v350.exe). Also here is non-https link which can by use to download it direcly in VM.

```
http://files.emulace.cz/demos/3dmark03v350.exe
```
